react-native-sdk/update_dependencies.js

```diff
@@ -0,0 +1,54 @@
+const fs = require('fs');
+
+const packageJSON = require('../../package.json');
+
+const RNSDKpackageJSON = require('./package.json');
+
+/**
+ * Updates dependencies from the app package.json with the peer dependencies of the RNSDK package.json.
+ */
+
+function updateDependencies() {
+    let updated = false;
+    let sortedDeps;
+
+    for (const key in RNSDKpackageJSON.peerDependencies) {
+        if (!packageJSON.dependencies.hasOwnProperty(key)) {
+            packageJSON.dependencies[key] =
+                '*' || RNSDKpackageJSON.peerDependencies[key];
+
+            updated = true;
+        }
+
+        sortedDeps = Object.keys(packageJSON.dependencies)
```

Comment:
I think you can simplify this by using the 2nd argument in JSON.stringify: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/JSON/stringify